packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/ReactAccessibilityDelegate.kt

```diff
@@ -224,25 +227,16 @@ public open class ReactAccessibilityDelegate( // The View this delegate is attac
     if (action == AccessibilityNodeInfoCompat.ACTION_EXPAND) {
       host.setTag(R.id.accessibility_state_expanded, true)
     }
-    if (action == AccessibilityNodeInfoCompat.ACTION_CLICK) {
```

Comment:
Not sure how these are generating events on the Mobile but not on the TV. I think the same outcome could be achieved by relying on `onAccessibilityEvent` and emitting press/longpress events on the JS layer accordingly.

Reply:
Confirmed, it is possible to have press and longPress handlers fire by setting accessibility props correctly, see below.